packages/suite-desktop-ui/src/support/DesktopUpdater.tsx

```diff
@@ -90,41 +86,41 @@ export const DesktopUpdater = ({ children }: DesktopUpdaterProps) => {
         }
 
         // Non visible states
-        if ([UpdateState.Checking, UpdateState.NotAvailable].includes(desktopUpdate.state)) {
+        if ([UpdateState.Checking, UpdateState.NotAvailable].includes(desktopUpdateState)) {
             return false;
         }
 
         const isEarlyAccessState = [
             UpdateState.EarlyAccessDisable,
             UpdateState.EarlyAccessEnable,
-        ].includes(desktopUpdate.state);
+        ].includes(desktopUpdateState);
 
         // Enable to setup Early Access even after updater error (when desktopUpdate.latest is undefined).
         if (!isEarlyAccessState && !desktopUpdate.latest) {
             return false;
         }
 
         return true;
-    }, [desktopUpdate.modalVisibility, desktopUpdate.state, desktopUpdate.latest]);
+    }, [desktopUpdate.modalVisibility, desktopUpdateState, desktopUpdate.latest]);
 
     const getUpdateModal = () => {
-        switch (desktopUpdate.state) {
+        switch (desktopUpdateState) {
             case UpdateState.EarlyAccessEnable:
                 return <EarlyAccessEnable hideWindow={hideWindow} />;
             case UpdateState.EarlyAccessDisable:
                 return <EarlyAccessDisable hideWindow={hideWindow} />;
             case UpdateState.Available:
                 return (
                     <Available
-                        hideWindow={hideWindow}
+                        onCancel={hideWindow}
                         latest={desktopUpdate.latest}
-                        isCancelable={isSettingsRoute}
```

Comment:
This was here because closing the modal during update by clicking outside, by clicking X or hitting ESC could lead to confusion - does downloading continue or not? See https://github.com/trezor/trezor-suite/issues/6655.